### ISSUE TITLE
VWAP indicator is wrong

### ISSUE THREAD

**Issue by Datamined** [bukosabino/ta#262]
In accordance with [the definition of VWAP](https://school.stockcharts.com/doku.php?id=technical_indicators:vwap_intraday) that you provide in the documentation VWAP should be calculated using cumulative volume, which is calculated from the start of the trading session (when we talk about relatively small timeframes, at least less than a day). There is no any rolling window in the definition, all volume since the day start (or other anchor) should be taken into account.

In other words, for each index this "rolling window size" should be equal to the index.
For instance, see the implementation [here](https://stackoverflow.com/a/29305456/7310209).

**Comment by zwangz** [comment]
I second. Just tried the library and VWAP was first indicator I used, immediately realised something was not right....

**Comment by mrsmirf** [comment]
I third this issue. This indicator is completely wrong.
